build/quicklink.php

```diff
@@ -71,6 +71,9 @@ function quicklink_enqueue_scripts() {
 
 			// Do not preload wp-content items (like downloads).
 			preg_quote( wp_parse_url( content_url(), PHP_URL_PATH ), '/' ),
+
+			// Do not preload links with get parameters.
+			'.*\?((.*=.*)(&?))+',
```

Comment:
This  can be simplified:

```suggestion
			'.*\?.+',
```

This should also account for cases where the query param has no value, like `/?foo`